nano/secure/store_partial.hpp

```diff
@@ -135,7 +139,7 @@ class store_partial : public store
 
 protected:
 	nano::ledger_constants & constants;
-	int const version_number{ 21 };
+	int const version_number{ 22 };
```

Comment:
As this table doesn't affect the way the ledger is organized it as it'll be disabled by default, do we want to increase the database version @clemahieu ?